### DISCUSSION TITLE
[Bug] Markdown comments are rendered in mobile notification feed

### DISCUSSION THREAD

**Discussion by JLLeitschuh** [community/community#7635] [Mobile] [answered]
Normally unrendered markdown comments are being rendered in the notification feed. This seems like a bug IMHO.

![DEDDB942-4563-43DE-A021-42B455255F11](https://user-images.githubusercontent.com/1323708/141473339-4263d65f-96e3-4e15-8731-6f5cd3e6f203.jpeg)

**Comment by brianlovin** [comment, marked as answer]
Thanks for the heads up @JLLeitschuh — the team is looking into how we might be able to clean up this experience, by at least removing HTML comments from these previews. It could get tricky with other inline markdown, but we'll see what's possible 😊
